docs/k8s/guides/local-cluster.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Local Cluster Options
+pagination_next: k8s/installation/install
+---
+
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
+# Run a local Kubernetes Cluster
+
+When installing the ngrok Kubernetes operator, you may want to install it in a locally running Kubernetes cluster rather than paying for one from a cloud provider.
+
+This is useful for:
+
+- Quickly getting started trying out the ngrok Operator
+- Local development & testing
+- Connecting containerized workloads on your local machine to your ngrok infrastructure
+- Faster iteration cycles without cloud latency
+
+Running Kubernetes inside containers has the benefits of being quick and simple to setup compared to a bare-metal installation and keeps the Kubernetes environment more isolated from the dependencies and other applications on your computer.
+This is not a recommendation for how you should productionize your Kubernetes setup, but rather a guide on getting an easy to use local environment setup for development and testing.
+The following sections outline various options for running Kubernetes inside Docker containers on your local machine, but you can run the ngrok Kubernetes operator in any Kubernetes cluster you like.
+
+## 1. Pick a Container Runtime
+
+Before setting up a local Kubernetes cluster, you need a container runtime. The container runtime is responsible for running containers, managing their lifecycle, and handling networking.
```

Comment:
This is a thought from @nijikokun: It might be useful to suggest a container runtime based on the cluster provider the user has, since they depend on each other

Reply:
Not sure that I follow. If the user already has a cluster provider installed, then they should already have a container runtime.

Reply:
I'm not sure what he meant exactly either, but he mentioned it in a sync with me and I thought I'd flag it. Maybe Niji can circle back and make a PR to add whatever changes he meant after this is merged